website/content/docs/concepts/transform.mdx

```diff
@@ -2,36 +2,125 @@
+
+<ImageConfig hideBorder>
+
+![Tokenization](/img/vault-tokenization-transformation-1.png)
+
+</ImageConfig>
+
+Vault protects a cryptographic mapping of tokens and plaintext values in its internal storage. The tokenization transform prevents attackers from recovering the plaintext, even if they steal the underlying transformation key and mapping values from Vault.
+
+Since tokenization is stateful, the encode operation must perform writes to storage on the primary node. As a result, the primary node's storage performance limits the scalability of the encode operation. This differs from Transit encryption and FPE where the encode operation does not require write to storage and can be horizontally scaled using performance standby nodes.
+
+By default, Vault stores the cryptographic mapping of tokens within its internal storage. For high-performance use cases, we recommend that you configure Vault to store this mapping in an external database. Users often prefer external stores because they can achieve a much higher performance scale and reduce the load on Vault's internal storage. Vault currently supports the following databases: PostgreSQL, MySQL, and MSSQL. For more information on external storage, visit the [Tokenization transform storage](/vault/docs/secrets/transform/tokenization#storage) documentation. 
+
+Similar to Transit encryption and FPE, tokenization requires keys and their management. Vault creates a new key for each tokenization transformation. This allows you to ensure a strong cryptographic distinction between different tokenization use cases. For example, a credit card processor might have different merchants for which it would like to tokenize credit card numbers, such that there is a strong cryptographic distinction between even the same credit card number used by different merchants. 
+
+Tokenization transform supports automatic key rotation based on a user-defined time interval. Each configured tokenization transformation keeps a set of versioned keys, such that when a key rotates, older key versions are still available for decoding tokens generated in the past. You can also specify the minimum key version Vault can use to decode value. Vault cannot decode generated tokens with keys below the minimum key version.
+
+#### Convergent tokenization
+
+By default, tokenization transformation produces a unique token for every encode operation. The Vault tokenization solution also supports convergent tokenization, which allows you to have the same encoded value for a given input so that you can query your database to count the number of entries for a given token.
+
+Convergent tokenization is useful when you want to do statistical analysis of the tokens as they relate to some other field in a database (without decoding the token), or if you need to tokenize in two different systems but be able to relate the results.
```

Comment:
```suggestion
<Highlight title="Convergent tokenization">

  By default, tokenization produces a unique token for every encode operation.
  Vault transform supports **convergent tokenization**, which lets you use the
  same encoded value for a given input.

  Convergent tokenization lets you perform statistical analysis of the tokens
  in your system **without decoding the token**. For example, counting the
  entries for a given token, querying relationships between a token and other
  fields in your database, and relating information that is tokenized across
  multiple systems.

</Highlight>
```

This isn't really a new section, it's more a tip for using tokenization effectively